modules/app.xqm

```diff
@@ -383,7 +383,7 @@ declare function app:package-group-to-list-item($package-group as element(packag
                                 for $change in $newest-package/changelog/change
                                 let $version := $change/@version/string()
                                 let $comment := $change/node()
-                                order by $version descending
+                                order by $version descending collation "http://www.w3.org/2013/collation/UCA?numeric=yes"
```

Comment:
I think you could get even better results by using `semver:compare` and `fn:sort`: https://github.com/eXist-db/semver.xq/blob/master/src/main/xquery/semver.xqm#L318`

The problem with a numeric sort is that it does suspect it does not respect ordering between non-numeric semver components like build labels etc.

Reply:
Agreed, it was just an interim measure. I plan to extend semver:sort to work on arbitrary items by adding a function parameter to find each item’s version string. When I have that I’ll go back here and swap that in.